src/olympia/scanners/models.py

```diff
@@ -417,14 +417,3 @@ class Meta(AbstractScannerResult.Meta):
 class ScannerQueryMatch(ModelBase):
     result = models.ForeignKey(ScannerQueryResult, on_delete=models.CASCADE)
     rule = models.ForeignKey(ScannerQueryRule, on_delete=models.CASCADE)
-
-
-class VersionScannerFlags(ModelBase):
```

Comment:
followup issue for the migration to drop the model/table plz.

Reply:
https://github.com/mozilla/addons-server/issues/14499